src/apply.c

```diff
@@ -2724,9 +2724,9 @@ use_stone(struct obj *tstone)
              * touchstone will yield the most */
             flint_made += 2;
         }
-        pline("You bang %s%s on %s.", ((obj->quan > 1L) ? "one of " : ""),
-              the(xname(obj)), the(xname(tstone)));
-        pline("It crumbles.");
+        You("bang %s%s on %s.", ((obj->quan > 1L) ? "one of " : ""),
+            the(xname(obj)), the(xname(tstone)));
+        pline_The("rock crumbles%s.", flint_made > 0 ? "" : " slightly");
```

Comment:
This is a oversight in the initial implementation of this feature. The rock should crumble completely even if no flint is made.

Reply:
This makes sense -- we had a conversation about the intended behavior a while back and I think I misunderstood what you said (you talked about an issue with the current way it works, something like "if the flint doesn't break you can just keep trying until you get some flint", and I thought you were describing the intended behavior).  This makes more sense to me.